package.json

```diff
@@ -12,6 +12,10 @@
   "bin": {
     "run": "bin/run.js"
   },
+  "scripts": {
+    "build": "babel src/ --out-dir lib/",
+    "test": "mocha"
```

Comment:
I'm up for removing `run` from testing, true we can have more precise feedback in tests if we will have more unit tests then it will work. Although I'm still thinking if we want to separate testing from build, I had couple of situations when I was testing compiled version which was not in sync with es6 version (I just forgot to build it).